apps/emqx_management/src/emqx_mgmt_cli.erl

```diff
@@ -344,12 +344,9 @@ modules(["unload", Name]) ->
     end;
 
 modules(["reload", "emqx_mod_acl_internal" = Name]) ->
-    case emqx_modules:reload(list_to_atom(Name)) of
-        ok ->
-            emqx_ctl:print("Module ~s reloaded successfully.~n", [Name]);
-        {error, Reason} ->
-            emqx_ctl:print("Reload module ~s error: ~p.~n", [Name, Reason])
-    end;
+    ok = emqx_modules:reload(list_to_atom(Name)),
```

Comment:
I had a closer look in all other callers of `emqx_module:reload/1`,
it seems to me that the code should not discard `{error, Reason}` and alway return `ok`.

sorry if I gave misleading info earlier.